packages/mermaid/src/diagrams/git/gitGraphRenderer.js

```diff
@@ -275,32 +275,36 @@ const drawCommits = (svg, commits, modifyGraph) => {
         }
       }
       if (commit.tag) {
-        const rect = gLabels.insert('polygon');
-        const hole = gLabels.append('circle');
-        const tag = gLabels
+        const g = gLabels.append('g');
+        if (gitGraphConfig.rotateTagLabel && dir === 'LR') {
+          g.attr('transform', 'translate(27,1) rotate(-45, ' + pos + ',' + y + ')');
```

Comment:
Use template string instead of concatenation.

Reply:
I removed this change because it was related to another feature (turning the tags).
I will probably open another PR for that.